techniques/systemSettings/misc/clockConfiguration/4.0/clockConfiguration.st

```diff
@@ -69,198 +84,131 @@ bundle agent check_clock_configuration
 
       "ntp_config_file_exists" expression => fileexists("/etc/ntp.conf");
 
-    clock_vardef.windows::
-      # check if we need to change the windows timezone
-      "need_to_change_timezone" not => strcmp("${windows_timezone}", "${current_timezone}");
-
-    any::
       "pass3" expression => "pass2";
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+    pass3::
+      "localtime_kept"     expression => "systemd.systemd_ntp_enabled_ok.!need_to_modify_timezone";
```

Comment:
this should not be set if clock_timezone_nochange - otherwise you have two reports:
E| compliant     ntpConfiguration          Time zone                                    Time zone dontchange was already correctly configured
E| n/a           ntpConfiguration          Time zone                                    Time zone edition disabled